pkg/storage/couchdb/couchdbstore.go

```diff
@@ -185,78 +203,39 @@ func (c *CouchDBStore) Put(k string, v []byte) error {
 
 	revID, err := c.getRevID(k)
 	if err != nil {
-		return err
+		if !errors.Is(err, storage.ErrValueNotFound) {
```

Comment:
these two conditions can be combined into a single one: 

```go
if err != nil && !errors.Is(err, storage.ErrValueNotFound) {
    ...
}
```

Reply:
Done